dev/tools/gen_defaults/lib/app_bar_template.dart

```diff
@@ -107,12 +101,6 @@ class _LargeScrollUnderFlexibleConfig with _ScrollUnderFlexibleConfig {
   TextStyle? get expandedTextStyle =>
     ${textStyle('md.comp.top-app-bar.large.headline')}?.apply(color: ${color('md.comp.top-app-bar.large.headline.color')});
 
-  @override
-  EdgeInsetsGeometry? get collapsedTitlePadding => const EdgeInsetsDirectional.only(start: 40);
-
-  @override
-  EdgeInsetsGeometry? get collapsedCenteredTitlePadding => const EdgeInsetsDirectional.only(start: 40);
-
```

Comment:
Changes in app_bar_template.dart are coming from a previous PR which only removed the defaults in app_bar.dart but didn't update the template. To avoid this kind of mismatch in the future, I will add a test shard to CI to make sure the template and generated defaults can match each other.